homeassistant/components/asuswrt/__init__.py

```diff
@@ -45,6 +50,8 @@
                 vol.Optional(CONF_SENSORS): vol.All(
                     cv.ensure_list, [vol.In(SENSOR_TYPES)]
                 ),
+                vol.Optional(CONF_INTERFACE, default=DEFAULT_INTERFACE): cv.string,
+                vol.Optional(CONF_DNSMASQ, default=DEFAULT_DNSMASQ): cv.string,
```

Comment:
Please use cv.isdir in this case to make sure that the specified path is a directory.